Bencodex.Tests/Types/BinaryTest.cs

```diff
@@ -57,6 +58,40 @@ public void ConstructorTakingString()
             Assert.Equal(hello, fromString);
         }
 
+        [Fact]
+        public void FromHex()
```

Comment:
The specification doesn't entirely rule out the uppercased hexadecimal, and I think they are common enough to be tested against.

Reply:
Now upper hexadecimals are tested too!